src/styles/legacy_main.css

```diff
@@ -25,7 +25,6 @@ body,
 }
 
 body {
-    background-color: var(--white);
```

Comment:
removing this line causes the whole app to become like this:

<img width="1792" alt="Screen Shot 2020-06-03 at 11 46 17" src="https://user-images.githubusercontent.com/1082127/83651429-d7f0a480-a58f-11ea-8dd2-2e7a4512fb19.png">

Reply:
Whoops...
Rolled it back.